src/libs/modulegroups.c

```diff
@@ -1902,6 +1930,26 @@ void init_presets(dt_lib_module_t *self)
   _preset_retrieve_old_presets(self);
 }
 
+static gchar *_presets_get_minimal(dt_lib_module_t *self)
+{
+  const gboolean is_modern = dt_conf_is_equal("plugins/darkroom/chromatic-adaptation", "modern");
```

Comment:
This is not used here. Can be removed.

Reply:
In fact no clean-up needed as this boolean is used by the SQA() macro.

Reply:
yes, I've removed it at first before gcc complain about SQA() ;)